### PR TITLE
Surface workflow finished time in list run API

### PR DESCRIPTION
**Pull request by IronPan**
/assign @rileyjbauer

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1122)

### CODE REVIEW

File: backend/src/apiserver/storage/run_store.go [hunk 1]
```diff
@@ -301,29 +303,30 @@ func (s *RunStore) CreateRun(r *model.RunDetail) (*model.RunDetail, error) {
...
-			"ScheduledAtInSec":        r.ScheduledAtInSec,
-			"Conditions":              r.Conditions,
-			"WorkflowRuntimeManifest": r.WorkflowRuntimeManifest,
-			"PipelineRuntimeManifest": r.PipelineRuntimeManifest,
-			"PipelineId":              r.PipelineId,
-			"PipelineSpecManifest":    r.PipelineSpecManifest,
-			"WorkflowSpecManifest":    r.WorkflowSpecManifest,
-			"Parameters":              r.Parameters,
-		}).ToSql()
+		"UUID":                    r.UUID,
```
Comment (rileyjbauer) [inline backend/src/apiserver/storage/run_store.go]: Are all these whitespace changes intentional?

File: backend/src/common/util/workflow.go [hunk 2]
```diff
@@ -135,6 +135,14 @@ func (w *Workflow) ScheduledAtInSecOr0() int64 {
 	return 0
 }
 
+func (w *Workflow) FinishedAt() int64 {
+	if w.Status.FinishedAt.IsZero(){
```
Comment (rileyjbauer) [inline backend/src/common/util/workflow.go]: this doesn't need to be nil-checked?

Comment (IronPan) [reply backend/src/common/util/workflow.go]: no finishedAt is a struct not a pointer so it won't be nil. checking zero should be good.

File: backend/src/apiserver/storage/run_store.go (hunk 1 above)
Comment (IronPan) [reply backend/src/apiserver/storage/run_store.go]: reverted

**Pull request merged**